sandbox-test/e2e-tests/src/main/java/de/sandbox/openbankinggateway/sandbox/internal/SandboxApp.java

```diff
@@ -1,4 +1,4 @@
-package de.adorsys.openbankinggateway.sandbox.internal;
+package de.sandbox.openbankinggateway.sandbox.internal;
```

Comment:
Can we use something like `de.adorsys.opba.testsandbox` ?

Sooner or later we'll come to open source releases to maven central...